tests/test_SwiftAccountUsage.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+
+from mock import Mock, MagicMock, call, patch
+
+from requests.structures import CaseInsensitiveDict
+
+import prometheus_openstack_exporter as poe
+
+
+class TestSwiftAccountUsage(unittest.TestCase):
+
+    @patch('prometheus_openstack_exporter.SwiftAccountUsage._get_account_ring')
+    @patch('prometheus_openstack_exporter.requests')
+    @patch('prometheus_openstack_exporter.config')
+    def test__get_account_usage(self, _config, _requests, _SwiftAccountUsage__get_account_ring):
+        _config = MagicMock()  # noqa: F841
+        _requests = MagicMock()  # noqa: F841
+        _SwiftAccountUsage__get_account_ring = MagicMock()  # noqa: F841
```

Comment:
I don't understand what you are doing here. `@patch` replaces the named reference with a MagicMock, and passes the MagicMock in as a parameter to the method. You are then replacing the MagicMocks that were passed in with fresh MagicMocks, and then not using them; the tests are referencing the original MagicMocks put in place by `@patch`. I think the above three lines can be removed.

Reply:
Uh, well, me either! I've cleaned this up a bit.